03.asynchronous/callback.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+// エラー無し
+const dbNoError = new sqlite3.Database(":memory:", () => {
+  dbNoError.run(
+    "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+    () => {
+      dbNoError.run(
+        "INSERT INTO books(title) values(?)",
```

Comment:
`CREATE TABLE` のスタイルに合わせるならこの INSERT 文も以下のようなスタイルにしたほうがよいです。

```suggestion
        "INSERT INTO books (title) values(?)",
```

Reply:
承知しました！修正します。